Common/Data/Consolidators/RenkoConsolidator.cs

```diff
@@ -351,21 +351,16 @@ public void Scan(DateTime currentLocalTime)
         /// <param name="consolidated">The newly consolidated data</param>
         protected virtual void OnDataConsolidated(RenkoBar consolidated)
         {
-            var handler = DataConsolidated;
-            if (handler != null) handler(this, consolidated);
+            DataConsolidated?.Invoke(this, consolidated);
 
-            var explicitHandler = _dataConsolidatedHandler;
-            if (explicitHandler != null) explicitHandler(this, consolidated);
+            _dataConsolidatedHandler?.Invoke(this, consolidated);
 
             Consolidated = consolidated;
         }
 
         /// <summary>Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.</summary>
         /// <filterpriority>2</filterpriority>
-        public void Dispose()
-        {
-            _dataConsolidatedHandler = null;
-        }
+        public void Dispose() => _dataConsolidatedHandler = null;
```

Comment:
Hmm, seems like an existing bug, but I think we can also set `DataConsolidated = null` here as well. As currently written, we're only clearing out event handlers on the `IDataConsolidator.DataConsolidated` event and not the well-typed `RenkoConsolidator.DataConsolidated` event.